src/libraries/System.Linq.Expressions/tests/Dynamic/InvokeMemberBindingTests.cs

```diff
@@ -54,7 +54,7 @@ public static IEnumerable<object[]> ObjectArguments()
             yield return new[] {new object()};
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/55070", typeof(PlatformDetection), nameof(PlatformDetection.IsLinqExpressionsBuiltWithIsInterpretingOnly))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55070", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
This test is passing with CoreCLR and IL compiler disabled, so whatever this is, it's iDevice-specific. This test was disabled in #54970 and not part of the mechanical refactoring.